.github/workflows/lint.yml

```diff
@@ -23,6 +23,10 @@ jobs:
         with:
           node-version: "14"
 
+      - name: Install JS dependencies
+        run: npm i
```

Comment:
This can be replaced with `npm ci` I think?

Reply:
Indeed it can! :P